.gitignore

```diff
@@ -6,6 +6,7 @@ __pycache__
 /env*/
 docs/build/
 docs/source/_build
+mypyc/doc/_build
```

Comment:
This was the default directory generated from running `make html` inside `mypyc/doc/`.